code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -431,6 +431,19 @@
 		/obj/item/weapon/melee/sabre
 		)
 
+
+
+/obj/item/weapon/storage/belt/sabre/AltClick(mob/user)
+	if(contents.len)
+		var/obj/item/I = contents[1]
+		user.visible_message("[user] takes [I] out of [src].", "<span class='notice'>You take [I] out of [src].</span>",\
+		)
+		user.put_in_hands(I)
+		update_icon(I)

```

Comment:
```update_icon()``` for the sabre BELT (which is the one you're calling with this line) takes no arguments, so don't pass it  ```I```